setup.py

```diff
@@ -56,7 +56,9 @@ def required(requirements_file):
     author_email='devs@mycroft.ai',
     url='https://github.com/MycroftAI/mycroft-core',
     description='Mycroft Core',
-    install_requires=required('requirements.txt'),
+    install_requires=required('requirements/requirements.txt'),
+    extras_requires=required('requirements/requirements-optional.txt'),
```

Comment:
Should this be [extras_require](https://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-extras-optional-features-with-their-own-dependencies)